src/include/utils.hpp

```diff
@@ -428,4 +428,44 @@ namespace utils
         std::transform(result.begin(), result.end(), result.begin(), tolower);
         return result;
     }
+
+    void hexToRgb(const std::string &hex, int &r, int &g, int &b)
+    {
+        std::stringstream ss;
+        ss << std::hex << hex.substr(1);
+        unsigned int hexColor;
+        ss >> hexColor;
+        r = (hexColor >> 16) & 0xFF;
+        g = (hexColor >> 8) & 0xFF;
+        b = hexColor & 0xFF;
+    }
+
+    // Set console text color using RGB values
+    void setColor(int r, int g, int b)
+    {
+        HANDLE hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
+        CONSOLE_SCREEN_BUFFER_INFOEX info;
+        info.cbSize = sizeof(CONSOLE_SCREEN_BUFFER_INFOEX);
+        GetConsoleScreenBufferInfoEx(hConsole, &info);
+
+        // Modify color table at index 1
+        info.ColorTable[10] = RGB(r, g, b);
+
+        SetConsoleScreenBufferInfoEx(hConsole, &info);
+        SetConsoleTextAttribute(hConsole, 10);
+    }
+
+    // Wrapper function to set color using hex code
+    bool setColor(const std::string &hexColor)
```

Comment:
This function never returns `false`. Why don't we return `void`?
```suggestion
    void setColor(const std::string &hexColor)
```